### DISCUSSION TITLE
need help with finding the zoom call recording

### DISCUSSION THREAD

**Discussion by orduck31** [cu-esiil-edu/stars-2024#26] [General]
@nquarder would you happen to have the zoom call with the recording of how to add a map to my profile? I looked through my emails and I don't have that one.  Could you please send it to me.  Thank you

**Comment by nquarder** [comment]
Hey @orduck31! Great discussion post question - 

Here is a link to the recording where we walk through how to add a map to your GitHub portfolio page:

https://cuboulder.zoom.us/rec/share/G_tNGmXEg8LSXClyEDOfgSElOb1ix_r6dPBIE0kEQZv0XPZqxtATdORtXlvcD71z.KQUeklU390WwEYGq?startTime=1714241375000

Let us know if you have any questions - and keep up the great work! :) -NQ
